package/kubectl-shell.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+CONTEXT_DIR=/var/lib/kubectl-contexts/$(uuidgen -r)
+mkdir -p ${CONTEXT_DIR}
+mount --bind ${CONTEXT_DIR} ${CONTEXT_DIR}
```

Comment:
Seemed necessary based on the docs for `unshare`, will remove and see if it's actually needed.